ckan/templates/activity_streams/activity_stream_email_notifications.text

```diff
@@ -1,7 +1,7 @@
 {% set num = activities|length %}{{ ungettext("You have {num} new activity on your {site_title} dashboard", "You have {num} new activities on your {site_title} dashboard", num).format(site_title=g.site_title, num=num) }} {{ _('To view your dashboard, click on this link:') }}
 
-{{ g.site_url + '/dashboard' }}
+{{ h.generate_url_to_root_path() + '/dashboard' }}
```

Comment:
We should always use `h.url_for()` as it contains the logic to handle the root_path correctly. Can you try this?
```suggestion
{{ h.url_for('dashboard.index') }}
```